README.md

```diff
@@ -134,6 +135,8 @@ The gateway communicates through SPI, ensure that SPI in enabled on the pi.
 
 To avoid capturing duplicate data, set the data capture frequency equal to or less than the expected uplink interval.
 
+If the error `ERROR: Failed to set SX1250_0 in STANDBY_RC mode` is seen in logs, unplug the raspberry pi for a few minutes and then try again.
```

Comment:
I wouldn't expect a user to hit this once this change is in but added here just in case.

Reply:
would it be possible for our logs to also return this suggestion?

Reply:
yep, added!